backend/services/eventService.js

```
@@ -0,0 +1,273 @@
+/**
+ * Find all events
+ * @returns {Promise<Array>} Array of Event objects
+ */
+const getAllEvents = async () => {
+    try {
+        return await Event.find()
+            .populate('club', 'name uuid')
+            .sort({ eventStart: 1 })
+            .lean();
+    } catch (error) {
+        throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)